src/core/resolvers/require.zig

```diff
@@ -111,7 +114,16 @@ pub fn zune_require(L: *VM.lua.State) !i32 {
                 break;
         }
     }
-    source = ar.source;
+    sourceConst = ar.source;
+    var source: ?[]u8 = null;
+    defer if (source) |r| allocator.free(r);
+
+    // normalize source to use unix path seps
+    if (sourceConst != null) {
+        source = try allocator.alloc(u8, sourceConst.?.len);
+        @memcpy(source.?, sourceConst.?);
```

Comment:
could be simplified to
```suggestion
        source = try allocator.dupe(u8, sourceConst);
```